index.html

```diff
@@ -18,6 +18,14 @@
             <li>About</li>
             <li>Contact</li>
         </ul>
+        <div class="nav_expand" id="hide">
+            <ul class="expand_ims">
+            <li><img class="iconx" src="./img/x.svg" alt="icon"></li>
+            <li id="ims"><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+            <li><a href="#port">Portfolio</a></li>
```

Comment:
- [ ] Please implement the changes that were required by the previous code reviewer as when the click on one of the menu items the mobile menu should disappear.